### ISSUE TITLE
php7 notice in WP_DEBUG mode

### ISSUE THREAD

**Issue by AiratHalitov** [bonny/WordPress-Simple-History#153]
Hi!
Thank you for good plugin!

I found some bugs today in WP_DEBUG mode on page:  `/wp-admin/index.php?page=simple_history_page`

### Log
```
[10-Sep-2018 21:20:14 UTC] $date_html, referer: https://test.com/wp-admin/index.php?page=simple_history_page
[10-Sep-2018 21:20:14 UTC] <span class='SimpleHistoryLogitem__permalink SimpleHistoryLogitem__when SimpleHistoryLogitem__inlineDivided'><a class='' href='https://test.com/wp-admin/index.php?page=simple_history_page#item/1'><span title="11.09.2018 - 02:06 местное время (10.09.2018 - 21:06 GMT)">02:06 (<time datetime="2018-09-10T21:06:50+00:00" class="SimpleHistoryLogitem__when__liveRelative">13 минут назад</time>)</span></a></span>, referer: https://test.com/wp-admin/index.php?page=simple_history_page
...
```
Similar two lines are repeated if there are many events.
Can you fix this? 

### One more question:

I cleared **Simple History** log but I still can see events on **Stats** graph (see screenshot):
![sh](https://user-images.githubusercontent.com/4050715/45325468-5b6f9400-b56a-11e8-9b82-1b047c288c69.png)
And loggers in debug tab of settings of plugin are not null
URL: `/wp-admin/options-general.php?page=simple_history_settings_menu_slug&selected-tab=debug`
Is it normal behavior? 

### Configuration:
* WordPress 4.9.8
* Simple History 2.28
* PHP 7.1.20-1+ubuntu16.04.1+deb.sury.org+1

**Comment by bonny** [comment]
Thanks for the report, there was a debug message that I forgot to remove.

Regarding the stats I think it's cache issue. It should go away after a while. Let me know if it's still there after a day or so.

**Comment by AiratHalitov** [comment]
> Regarding the stats I think it's cache issue. It should go away after a while. Let me know if it's still there after a day or so.

Yes, now stats looks good. 
Thanks!

**Comment by AiratHalitov** [comment]
> Thanks for the report, there was a debug message that I forgot to remove.

Fixed in v2.28.1
Thanks!
